### PR TITLE
Fix the `check` target on MSVC

### PR DESCRIPTION
**Pull request by DinCahill**
On MSVC we need `ctest -C Release` to run the tests.

### CODE REVIEW

**Comment by DinCahill** [comment]
I'll add some more to this PR in a bit.

**Comment by DinCahill** [comment]
Now, in `WindowsBuilder.ps1`, `-playd` will only build playd, without the tests. `-tests` implies `-playd`, and `-check` implies `-tests`. I could've done it some other way, but it's more effort. At least playd and the tests are built concurrently.

I've also added notes that Git and 7-Zip are required for building the dependencies.

**Comment by LordAro** [comment]
:+1:

File: WindowsBuilder.ps1 [hunk 1]
```diff
@@ -155,10 +179,18 @@ function BuildPlayd ($arch, $archdir, $build) {
         "x86" { $cmake_generator = "Visual Studio 14 2015"; $msbuild_platform = "Win32" }
         "x64" { $cmake_generator = "Visual Studio 14 2015 Win64"; $msbuild_platform = "x64" }
     }
+	$targets = "playd"
+	if ($tests -Or $check) {
+		$targets="$targets;playd_tests"
+	}
```
Comment (LordAro) [inline WindowsBuilder.ps1]: Some alignment fun here

Comment (DinCahill) [inline WindowsBuilder.ps1]: thx

File: WindowsBuilder.ps1 [hunk 2]
```diff
@@ -155,10 +179,18 @@ function BuildPlayd ($arch, $archdir, $build) {
         "x86" { $cmake_generator = "Visual Studio 14 2015"; $msbuild_platform = "Win32" }
         "x64" { $cmake_generator = "Visual Studio 14 2015 Win64"; $msbuild_platform = "x64" }
     }
+        $targets = "playd"
+        if ($tests -Or $check) {
+                $targets="$targets;playd_tests"
+        }
```
Comment (DinCahill) [inline WindowsBuilder.ps1]: Soz I just untabified and got rekt

**Pull request merged**